flang/lib/Optimizer/CodeGen/Target.cpp

```diff
@@ -1081,6 +1081,9 @@ struct TargetLoongArch64 : public GenericTarget<TargetLoongArch64> {
   using GenericTarget::GenericTarget;
 
   static constexpr int defaultWidth = 64;
+  static constexpr int GRLen = defaultWidth; /* eight bytes */
```

Comment:
I wonder how flang dealing with different ABIs?

Reply:
Thanks for your review. This patch only implements `mabi=lp64d` in LoongArch64, and does not currently support others,  though they may be supported in the future. To make it more explicit, I will modify this patch to report an error when other ABIs or conflicting options are detected.